e2e/wdio/core/tests/dynamic-page.e2e-spec.ts

```diff
@@ -0,0 +1,156 @@
+    });
+
+    it('should check collapsing area', () => {
+        checkCollapsingArea(tabsExample);
+        checkCollapsingArea(columnLayoutExample);
+        checkCollapsingArea(responsiveExample);
+    });
+
+    it('should check selecting tabs', () => {
+        click(tabsExample + button);
+        scrollIntoView(tabsContent, 1);
+        expect(getElementClass(tab, 1)).toContain('is-selected', 'tab is not highlited as selected after scroll to content');
+        expect(getElementClass(tab, 0)).not.toContain('is-selected', 'tab is selected, but should not');
+        expect(getElementClass(tab, 2)).not.toContain('is-selected', 'tab is selected, but should not');
+    });
+
+    it('should check changing size of buttons size', () => {
+        click(responsiveExample + button);
+        click(dynamicPage + button, 7);
+        const buttonsLength = getElementArrayLength(dynamicPage + button);
+        for (let i = 0; i < buttonsLength; i++) {
+            if (i !== 5 && i !== 6) {
+                expect(getElementClass(dynamicPage + button, i)).not.toContain('compact', 'size of the button still compact');
+            }
+        }
+        click(dynamicPage + button, 7);
+        for (let i = 0; i < buttonsLength; i++) {
+            if (i !== 5 && i !== 6) {
+                expect(getElementClass(dynamicPage + button, i)).toContain('compact', 'size of the button did not change to cozy');
+            }
+        }
+    });
+
+    it('should check 📍 button working', () => {
+        checkPiningArea(tabsExample);
+        checkPiningArea(tabsExample);
+        checkPiningArea(responsiveExample);
+    });
+
+    it('should check open-close flexible columns', () => {
+        click(columnLayoutExample + button);
+        click(dynamicPage + button, 7);
+        pause(1000);
+        expect(isElementDisplayed(flexileColumn, 1)).toBe(true, 'flexible column is not visible');
+        click(flexileColumn + button, 10);
+        pause(1000);
+        expect(isElementDisplayed(flexileColumn, 2)).toBe(true, 'flexible column is not visible');
+        click(flexileColumn + button, 11);
+        pause(1000);
+        expect(isElementDisplayed(flexileColumn, 0)).toBe(false, 'flexible column is not hidden');
+        expect(isElementDisplayed(flexileColumn, 1)).toBe(false, 'flexible column is not hidden');
+        expect(isElementDisplayed(flexileColumn, 2)).toBe(true, 'flexible column is not visible');
+        click(flexileColumn + button, 11);
+        pause(1000);
+        expect(isElementDisplayed(flexileColumn, 0)).toBe(true, 'flexible column is not visible');
+        expect(isElementDisplayed(flexileColumn, 1)).toBe(true, 'flexible column is not visible');
+        expect(isElementDisplayed(flexileColumn, 2)).toBe(true, 'flexible column is not visible');
+        click(flexileColumn + button, 12);
+        pause(1000);
+        expect(isElementDisplayed(flexileColumn, 2)).toBe(false, 'flexible column is not hidden');
+    });
+
```

Comment:
missing checks for page links
![2021-09-09](https://user-images.githubusercontent.com/47522152/132680397-594999fd-ceeb-4062-9cfe-71f26f439b80.png)